python/taichi/misc/np2ply.py

```diff
@@ -128,6 +125,19 @@ def add_vertex_rgba(self, r: np.array, g: np.array, b: np.array, a: np.array):
         self.add_vertex_channel("blue", "float", b)
         self.add_vertex_channel("Alpha", "float", a)
 
+    # pass ti vector/matrix tensor directly
+    def add_vertex_rgba(self, rgba):
+        if not isinstance(rgba, np.ndarray):

```

Comment:
could we add an assert here to specify what exactly type we're expecting 'rgba' to be?

Reply:
Sure, I would suggest `rgba` should either assert `np.ndarray` or 'ti.vector/Matrix'. Any other type would you suggest?